changelog.d/9428.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in v1.27.0: "TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType." related to the user directory.
```

Comment:
Does this have a user visible effect?

Reply:
I don't think so since the user directory wouldn't be ready to be updated anyway?

Reply:
Fair :)